jsonnet/rules.libsonnet

```diff
@@ -611,5 +611,42 @@ function(params) {
         },
       ],
     },
+    {
+      name: 'openshift-console.rules',
+      rules: [
+        {
+          expr: 'sum(console_auth_login_requests_total)',
+          record: 'cluster:console_auth_login_requests_total:sum',
+        },
+        {
+          expr: 'sum(console_auth_login_successes_total) by (role)',
+          record: 'cluster:console_auth_login_successes_total:sum',
+        },
+        {
+          expr: 'sum(console_auth_login_failures_total) by (reason)',
+          record: 'cluster:console_auth_login_failures_total:sum',
+        },
+        {
+          expr: 'sum(console_auth_logout_requests_total) by (reason)',
+          record: 'cluster:console_auth_logout_requests_total:sum',
+        },
+        {
+          expr: 'sum(console_usage_total) by (event, perspective)',
+          record: 'cluster:console_usage_total:sum',
+        },
+        {
+          expr: 'max(console_usage_users) by (role)',
+          record: 'cluster:console_usage_users:max',
+        },
+        {
+          expr: 'sum(console_plugins_info) by (name, state)',
+          record: 'cluster:console_plugins_info:sum',
+        },
+        {
+          expr: 'sum(console_customization_perspectives_info) by (name, state)',
+          record: 'cluster:console_customization_perspectives_info:sum',
```

Comment:
I understand that this is not something for this PR but maybe its possible to change without too much hassle. I would argue to rename the label `name` to `perspective`. Its more descriptive and used in other places as well.